### PR TITLE
hotfix: refactor man.dvc.org redirect again and add try to force no-cache

### PR DESCRIPTION
**Pull request by jorgeorpinel**
This time reduced code as much as possible, reintroduced ternary operator...

For #768

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -27,21 +27,18 @@ app.prepare().then(() => {
...
-        new_path = pathname
-      }
       res.writeHead(301, {
-        Location: 'https://dvc.org/doc/command-reference' + new_path
+        Location:
+          'https://dvc.org/doc/command-reference' +
+          (['/get-url', '/import-url'].indexOf(pathname) < 0
+            ? pathname.replace('-', '/')
+            : pathname)
       })
```
Comment (jorgeorpinel) [inline server.js]: This is not possible to test in the review app... However I tried it locally duplicating this else case and substituting `dvc.org` with `localhost:3000` and the redirects work correctly: from man.localhost:3000/get-url and /import-url to localhost:3000/doc/command-reference/get-url and /import-url respectively, but from any other man.localhost:3000 path with dashes (such as /cache-dir) to localhost:3000/doc/command-reference{path with slashes} (e.g. to .../command-reference/cache/dir).

Please try to test this locally before merging to see if this somehow fixes the redirect on prod.

File: server.js [hunk 2]
```diff
@@ -27,21 +27,27 @@ app.prepare().then(() => {
...
+      })
+      res.end()
+    } else if (req.headers.host === 'man.localhost:3000') {
+      res.writeHead(301, {
+        Location:
+          'http://localhost:3000/doc/command-reference' +
+          (['/get-url', '/import-url'].indexOf(pathname) < 0
+            ? pathname.replace('-', '/')
+            : pathname)
       })
```
Comment (jorgeorpinel) [inline server.js]: In fact I added this code temporarily! Please download branch and run `yarn dev` and verify that:

http://man.localhost:3000/get-url -> http://localhost:3000/doc/command-reference/get-url
http://man.localhost:3000/import-url -> http://localhost:3000/doc/command-reference/import-url
http://man.localhost:3000/cache-dir -> http://localhost:3000/doc/command-reference/cache/dir
http://man.localhost:3000/no-thing -> http://localhost:3000/doc/command-reference/no/thing (not found)

Comment (jorgeorpinel) [reply server.js]: > Actually http://man.localhost:3000/import-url sends me to **https**://localhost:3000/doc/command-reference/import-url right now, Idk why. This causes a `ERR_SSL_PROTOCOL_ERROR` on Chrome. Whatever this is it wouldn't be a problem on prod which does support HTTPs.

Comment (shcheklein) [reply server.js]: @jorgeorpinel please, see the last comment in the ticket.

Comment (jorgeorpinel) [reply server.js]: https://github.com/iterative/dvc.org/issues/768#issuecomment-549456704

^ Thanks for figuring the caching issue out. I was going crazy over here haha.

OK I'm reverting this commit with the man.localhost... redirect.

**Comment by shcheklein** [comment]
@jorgeorpinel merge this?

File: server.js [hunk 3]
```diff
@@ -25,23 +25,21 @@ app.prepare().then(() => {
...
-      let new_path
-      if (['/get-url', '/import-url'].indexOf(pathname) < 0) {
-        new_path = pathname.replace('-', '/')
-      } else {
-        new_path = pathname
-      }
       res.writeHead(301, {
-        Location: 'https://dvc.org/doc/command-reference' + new_path
+        'Cache-Control': 'no-cache',
+        Location:
```
Comment (jorgeorpinel) [inline server.js]: Added no-cache, let's merge and see what this does...

**Pull request merged**